biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -0,0 +1,1172 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>PDB ID</th>\n",
+       "      <th>Method</th>\n",
+       "      <th>Resolution</th>\n",
+       "      <th>Structure</th>\n",
+       "      <th>Ligand ID</th>\n",
+       "      <th>Ligand name</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>3SHC</td>\n",
+       "      <td>X-RAY DIFFRACTION</td>\n",
+       "      <td>1.90</td>\n",
+       "      <td>Human Thrombin</td>\n",
+       "      <td>B01</td>\n",
+       "      <td>D-phenylalanyl-N-[(4-chloropyridin-2-yl)methyl...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>3P17</td>\n",
+       "      <td>X-RAY DIFFRACTION</td>\n",
+       "      <td>1.43</td>\n",
+       "      <td>Thrombin</td>\n",
+       "      <td>99P</td>\n",
+       "      <td>D-phenylalanyl-N-(pyridin-3-ylmethyl)-L-prolin...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2ZNK</td>\n",
+       "      <td>X-RAY DIFFRACTION</td>\n",
+       "      <td>1.80</td>\n",
+       "      <td>Thrombin</td>\n",
+       "      <td>31U</td>\n",
+       "      <td>D-leucyl-N-(4-carbamimidoylbenzyl)-L-prolinamide</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  PDB ID             Method  Resolution       Structure Ligand ID  \\\n",
+       "0   3SHC  X-RAY DIFFRACTION        1.90  Human Thrombin       B01   \n",
+       "1   3P17  X-RAY DIFFRACTION        1.43        Thrombin       99P   \n",
+       "2   2ZNK  X-RAY DIFFRACTION        1.80        Thrombin       31U   \n",
+       "\n",
+       "                                         Ligand name  \n",
+       "0  D-phenylalanyl-N-[(4-chloropyridin-2-yl)methyl...  \n",
+       "1  D-phenylalanyl-N-(pyridin-3-ylmethyl)-L-prolin...  \n",
+       "2   D-leucyl-N-(4-carbamimidoylbenzyl)-L-prolinamide  "
+      ]
+     },
+     "execution_count": 37,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "thrombin_df.head(3)"
```

Comment:
I'd show the default output before adding numbers to the function here.

Reply:
Done